CMakeUserPresets.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": 3,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 26,
```

Comment:
Is this the CMake version needed for the android platform specified?

Reply:
I forgot to rebase... CMakeUserPresets.json shouldn't be commited into a repo.